australia/australia.md

```diff
@@ -6,4 +6,9 @@ Yggdrasil configuration file to peer with these nodes.
 ### Sydney
 
 * Sydney, Flowvps, sponsored by [bondarevv23](https://github.com/bondarevv23) operated by [EKCKABATOP54](androposhtar1029@gmail.com)
-  * `tcp://195.114.14.86:7676`
\ No newline at end of file
+  * `tcp://195.114.14.86:7676`
+
+### Adelaide
+
+* Adelaide, operated by [sirsegv](https://sirsegv.moe/)
+  * `tcp://[2001:44b8:26e:1300::1]`
```

Comment:
Missing port number?

Reply:
ahh yeah, I missed that somehow :sweat_smile: fixed now!